static/style.css

```
@@ -1,9 +1,12 @@
 body {
-	
+
     background-color: black;
+
 }
 
 #instrument {
+	padding: 10px 10px 10px 10px;
+	box-shadow: 0px 5px 10px 15px #ffffff;
```

```suggestion
	box-shadow: 0px 5px 10px 15px #ffffff;
```
I don't think it makes it look any more good. So, I'd prefer to avoid the `box-shadow`. Although, the `padding` looks good to me so I'd like to consider it. 
Please make the required changes and then we are good to go! :)

---

Furthermore, can you remove the individual paddings that I have provided in line 12 and 13? Adding a general padding will be overshadowed if we keep `padding-right` as well as `padding-top`. Also, ensure that everything is in proper indentation to ensure clarity and readability in the code :) 
The following is the link to the required changes in [padding](https://github.com/sonakshisaxena1/Musical-Tiles/blob/e626db26c820f13bc2d852449339fb1ade81e2cd/static/style.css#L12)